lang/cs/Org.Apache.REEF.Tests/Functional/Bridge/TestCloseTask.cs

```diff
@@ -322,6 +347,52 @@ public void OnError(Exception error)
             }
         }
 
+        private sealed class BreakTestTask : ITask, IObserver<ICloseEvent>
+        {
+            private bool _break;
+            private int _iterate = 1;
+
+            [Inject]
+            private BreakTestTask()
+            {
+            }
+
+            public byte[] Call(byte[] memento)
+            {
+                while (!_break && _iterate < 100)
```

Comment:
In case the task doesn't receive the close event, we don't want the task runs forever. The iterate can be local in this test.